tidb-cloud/migrate-sql-shards.md

```diff
@@ -0,0 +1,556 @@
+    ```
+
+    The parameters used in the command above are described as follows:
+
+    |Parameter              |Description    |
+    |-                      |-              |
+    |`--master-addr`        |The `{advertise-addr}` of any DM-master node in the cluster where `dmctl` is to be connected. For example: 192.168.11.110:9261|
+    |`operate-source create`|Loads the data source to the DM cluster.|
+
+    The following is an example output:
+
+    ```shell
+    tiup is checking updates for component dmctl ...
+
+    Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source1.yaml
+
+    {
+       "result": true,
+       "msg": "",
+       "sources": [
+           {
+               "result": true,
+               "msg": "",
+               "source": "mysql-replica-01",
+               "worker": "dm-192.168.11.111-9262"
+           }
+       ]
+    }
+
+    ```
+
+4. Run the following command in a terminal. Use `tiup dmctl` to load the second data source configuration into the DM cluster:
+
+    ```shell
+    [root@localhost ~]# tiup dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source2.yaml
+    ```
+
+    The following is an example output:
+
+    ```shell
+    tiup is checking updates for component dmctl ...
+
+    Starting component `dmctl`: /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl /root/.tiup/components/dmctl/v6.0.0/dmctl/dmctl --master-addr 192.168.11.110:9261 operate-source create dm-source2.yaml
+
+    {
+       "result": true,
+       "msg": "",
+       "sources": [
+           {
+               "result": true,
+               "msg": "",
+               "source": "mysql-replica-02",
+               "worker": "dm-192.168.11.112-9262"
+           }
+       ]
+    }
+    ```
+
+### Step 2. Create a replication task
+
+1. Create a `test-task1.yaml` file for the replication task. Configure the incremental replication mode and the starting point of the data source in the file.
+
+2. Find the starting point in the metadata file of MySQL instance1 exported by Dumpling. For example:
+
+    ```toml
+    Started dump at: 2022-05-25 10:16:26
+    SHOW MASTER STATUS:
+           Log: mysql-bin.000002
+           Pos: 246546174
+           GTID:b631bcad-bb10-11ec-9eee-fec83cf2b903:1-194801
+    Finished dump at: 2022-05-25 10:16:27
+    ```
+
+3. Find the starting point in the metadata file of MySQL instance2 exported by Dumpling. For example:
+
+    ```toml
+    Started dump at: 2022-05-25 10:20:32
+    SHOW MASTER STATUS:
+           Log: mysql-bin.000001
+           Pos: 1312659
+           GTID:cd21245e-bb10-11ec-ae16-fec83cf2b903:1-4036
+    Finished dump at: 2022-05-25 10:20:32
+    ```
+
+4. Edit the task configuration file `test-task1`, to configure the incremental replication mode and replication starting point for each data source.
+
```

Comment:
Do we need an indentation for the file content as follows?

Reply:
I prefer to leave it as is. Indentation will make a mess :S